ui/src/main/webapp/package.json

```diff
@@ -75,6 +75,7 @@
     "@types/node": "^6.0.38",
     "@types/prismjs": "^1.4.18",
     "@types/showdown": "^1.4.31",
+    "@types/d3": "4.4.1",
```

Comment:
Ok got it. d3 is the JS code, @types/d3 is the typescript stuff around that which needs to catch up.